website/docs/reference/widgets/table/README.md

```diff
@@ -1,36 +1,90 @@
-# Table
-
-The table widget displays data in rows and columns. You can display data from an API in a table, trigger an action when a user selects a row, and even work with sizable paginated data sets.
+---
+toc_min_heading_level: 2
+toc_max_heading_level: 3
+---
 
-<VideoEmbed host="youtube" videoId="-rzePEV2QQ8" title="How to use Table Widget?" caption="How to use Table Widget?"/>
+# Table
 
+The table widget displays data in rows and columns. You can display data from an API in a table, trigger an action when a user selects a row, and even work with sizable paginated data sets. Once a table is connected to a datasource, you can also use [inline-editing](/reference/widgets/table/inline-editing) features to make changes and additions to the table data directly in the widget.
 
-:::info
-If you already know the basics and want to learn how to display data and handle pagination inside a table, [Read this guide](/core-concepts/data-access-and-binding/displaying-data-read/display-data-tables#pagination).
-:::
+<VideoEmbed host="youtube" videoId="-rzePEV2QQ8" title="How to use Table Widget" caption="How to use Table Widget"/>
 
 ## Properties
```

Comment:
Please break down the table into multiple tables if you feel the information is too much for one table. But make sure you don't spread out the property table for eg: Styles. You can refer to the platform to see how the properties are being segregated

Reply:
I think the single table may be okay, since the properties appear in the same order as in the properties pane and users can use the browser Find function to help; breaking it into smaller tables might just make it more confusing. I'm open to alternative ways to try though, just let me know @danciaclara